### ISSUE TITLE
Enzyme localization on extracellular membrane & permeability

### ISSUE THREAD

**Issue by tkc5** [Open-Systems-Pharmacology/Forum#665]
Hello,

I am looking for guidance on how PK-sim handles enzyme localization on the extracellular membrane. I am attempting to model an alkaline phosphatase expressed on the extracellular membrane in the small intestine, where it would act on an orally dosed drug. From reading the documentation, I have implemented this enzyme as being expressed on the extracellular membrane (apical). To me, this should mean that the enzyme appears between the intestinal lumen and the intracellular space (mucosa cytosol). However, in my simulation output, the enzyme only shows up in the interstitial compartment of the small intestine (there is no option to view the concentration of this enzyme in the intracellular space, only the plasma or interstitial compartments). 

Why does PK-sim count this enzyme as being present in the interstitial space? Are these the appropriate localization settings for an enzyme to be expressed between the lumen and intracellular space, and if not, what is the recommended way to handle this? 

Additionally, the permeability of the dosed drug seems to have a significant impact on my result, even in the limit of an extremely active enzyme. I would have expected the permeability of the dosed drug to be relatively unimportant if the enzyme is catalyzing a reaction of the dosed drug from the extracellular membrane of the mucosa, ie before the drug enters any cell. Where does permeability come in when modeling interactions with enzymes on the extracellular membrane of the intestinal mucosa?

Thanks!
Theresa

**Comment by PavelBal** [comment]
Dear Theresa,

if by "I have implemented this enzyme as being expressed on the extracellular membrane (apical)" you are referring to this combination

![image](https://user-images.githubusercontent.com/25903040/103764143-38126200-501b-11eb-8b27-048fff82203e.png)

then "Apical" is valid for the expression in vascular endothelium.

If you want to have your enzyme expressed in the lumen, you have to select the respective values:

![image](https://user-images.githubusercontent.com/25903040/103764277-77d94980-501b-11eb-862c-fc6fef5201d5.png)


This may be not fully intuitive, but this is the only way to place the enzyme into the lumen. However be aware that for calculation of the final amount of the enzyme, the concentration of luminal space and not of mucosal tissue will be taken into account.

**Comment by tkc5** [comment]
Hi Pavel,

Thanks very much for this response. Yes, I see now that my comment about localization was unclear, but you're exactly right about what I did. I notice now that the relative expression in the vascular endothelium is 0, so the choice of apical or basolateral under localization does not matter. 

In my simulation, I already have the relative expression of the alkaline phosphatase set to 0 in the GIT - Lumen (that was the default setting). Is there any way to see in the simulation output that the enzyme is appearing in the right place? As I mentioned before, the only concentration of the enzyme I can find among the output options is the interstitial concentration. It is not intuitive to me to set the enzyme concentration in the lumen to 0 to place the enzyme in the lumen.

If the enzyme is properly expressed in the lumen, and the activity is made artificially very high, I would expect to see (almost full, in the limit of infinite enzyme activity) drug conversion in the lumen, and thus the simulation results would not be strongly dependent on the permeability of the dosed drug. However, I do observe strong dependence on the dosed drug permeability, which makes me question if I have selected the proper settings for the enzyme. Or, is there some other process I am unaware of? The main regions the enzyme is expressed, by default, are:

![image](https://user-images.githubusercontent.com/77025348/103774995-6b44f900-4ffb-11eb-8c87-a75e5556711f.png)
I have the expression in the lumen set to 0 as shown in your post.

If the enzyme is only present within the mucosa, and the drug must first enter the cell to encounter the enzyme, that could explain the behavior I am seeing in my simulation output.

Thanks!
